src/evaluation_system/misc/utils.py

```diff
@@ -526,16 +527,16 @@ class PrintableList(list):
 
     def __init__(self, *args, **kwargs):
         try:
-            self.seperator = kwargs.pop("seperator")
+            self.separator = kwargs.pop("separator")
```

Comment:
That is dangerous. Are you sure that the key that as been added to kwargs somewhere is also renamed?

Reply:
I think this typo was the reason that we got an error in one `unit-test` func. But i will take a note to have a deep look in next PR